src/styles/themes/thorin.css

```
@@ -0,0 +1,101 @@
+  --ethereum-identity-kit-shadow-small: 0 1px 2px rgba(16, 24, 40, 0.06);
+  --ethereum-identity-kit-shadow-medium: 0 4px 8px rgba(16, 24, 40, 0.08);
+  --ethereum-identity-kit-shadow-large: 0 12px 24px rgba(16, 24, 40, 0.12);
+
+  --ethereum-identity-kit-social-twitter: #5298ff;
+  --ethereum-identity-kit-social-github: #344054;
+  --ethereum-identity-kit-social-discord: #5865f2;
+
+  --ethereum-identity-kit-sign-in-button: #5298ff;
+  --ethereum-identity-kit-sign-in-button-hover: #3d84eb;
+
+  font-family: 'Satoshi', 'Inter', sans-serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Satoshi" (font-family-name-quotes)

(font-family-name-quotes)